src/data/api.js

```diff
@@ -5,14 +5,26 @@ import { ExamAction } from '../constants';
 const BASE_API_URL = '/api/edx_proctoring/v1/proctored_exam/attempt';
 
 export async function fetchExamAttemptsData(courseId, sequenceId) {
```

Comment:
Is there a reason that we don't have multi-line comments describing what each of these functions do as we do in the back-end?

Reply:
We typically don't leave comments for the api functions because they are essentially wrappers - they call an endpoint and return the data. Here's an example of a more mature api.js file: https://github.com/openedx/frontend-app-learning/blob/master/src/course-home/data/api.js